articles/purview/tutorial-using-python-sdk.md

```diff
@@ -0,0 +1,335 @@
+
+    Select the collection you want to work with, and go on the "Role assignments" tab. Add the service principal (using its name) in the following roles: Collection admins, Data source admins, Data curators, Data readers.
+
+## Install the Python packages
+1.	Open a new command prompt or terminal
+2.	Install the Azure identity package for authentication:
+    ```bash
+    pip install azure-identity
+    ```
+3.	Install the Azure Purview Scanning Client package
+    ```bash
+    pip install azure-purview-scanning
+    ```
+4.	Install the Azure Purview Client package
+    ```bash
+    pip install azure-purview-catalog
+    ```
+5.	Install the Azure core package
+    ```bash
+    pip install azure-core
+    ```
+
+## Instantiate a Scanning and a Catalog client 
+In this section, you learn how to instantiate:
+*    A scanning client useful to deal with scan-related tasks like registering data sources, creating and managing scan rules, triggering a scan, etc. 
+*    A catalog client useful to interact with the catalog through searching, browsing the discovered assets, identifying the sensitivity of your data, etc.
+
+First you need to authenticate to your Azure Active Directory. In this tutorial, you will use the [client secret you created](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal#option-2-create-a-new-application-secret).
+
+
+1.	Start with required import statements:
+    ```python
+    from azure.purview.scanning import PurviewScanningClient
+    from azure.purview.catalog import PurviewCatalogClient
+    from azure.identity import ClientSecretCredential 
+    from azure.core.exceptions import HttpResponseError
+    ```
+
+2.	As said previously, you need to specify the following information in the code: 
+    * Client ID (or Application ID)
+    * Tenant ID (or Directory ID)
+    * Client secret
+    
+    ```python
+    client_id = "<your client id>" 
+    client_secret = "<your client secret>"
+    tenant_id = "<your tenant id>"
+    ```
+
+3.	You also need to specify the name of your purview account:
+    ```python
+    reference_name_purview = "<name of your purview account>"
+    ```
+4.	You can now instantiate the two clients:
+    ```python
+    def get_credentials():
+        credentials = ClientSecretCredential(client_id=client_id, client_secret=client_secret, tenant_id=tenant_id)
+        return credentials
+
+    def get_purview_client():
+        credentials = get_credentials()
+        client = PurviewScanningClient(endpoint=f"https://{reference_name_purview}.scan.purview.azure.com", credential=credentials, logging_enable=True)  
+        return client
+    
+    def get_catalog_client():
+        credentials = get_credentials()
+        client = PurviewCatalogClient(endpoint=f"https://{reference_name_purview}.purview.azure.com/", credential=credentials, logging_enable=True)
+        return client
+    ```
+ 
+You are now able to do several operations in a similar fashion.
```

Comment:
Can you clarify this sentence? Are you going to show us how to do other things with a similar piece of code in the following sections? What kinds of things?

Reply:
I rephrased this sentence, is it clearer?

Reply:
Unfortunately, I'm not seeing any change to this line.